features/extra/no-mail.php

```diff
@@ -1,6 +1,7 @@
 <?php
 
-function wp_mail() {
-	// do nothing
+function wp_mail( $to ) {
+	// Log for testing purposes
+	WP_CLI::log( "WP-CLI test suite: Sent email to {$to}." );
```

Comment:
@srguglielmo Here's how I set up the tests, part one. `wp_mail()` writes to STDOUT...